kclvm/tools/src/LSP/Cargo.toml

```diff
@@ -25,6 +25,9 @@ kclvm-parser = {path = "../../../parser", version = "0.4.5"}
 kclvm-compiler = {path = "../../../compiler", version = "0.4.5"}
 kclvm-sema = {path = "../../../sema", version = "0.4.5"}
 kclvm-error = {path = "../../../error", version = "0.4.5"}
+kclvm-config ={ path = "../../../config", version = "0.4.5"}
+kclvm-cmd = { path = "../../../cmd", version = "0.4.5"}
+kclvm-runner = {path = "../../../runner", version = "0.4.5"}
```

Comment:
It is better not to rely on runner and cmd directly, but will introduce llvm codegen dependency additionally.